third-party/linshare/src/test/java/org/apache/james/linshare/LinshareExtension.java

```diff
@@ -109,6 +112,14 @@ public LinshareConfiguration configurationWithJwtFor(LinshareFixture.Credential
             .downloadShare(document.asString(), filename);
     }
 
+    public UserResult createUser(LinshareFixture.Credential credential, UserCreationRequest userCreationRequest) {
```

Comment:
This method SHOULD be named `createTechnicalAccount`

Reply:
When we create a technical user we always associate some permissions to it (because we cannot use it if we don't grant permissions).

I'd propose to merge the 2 methods into one so that API users don't forget to grant permissions. And bonnus: we will only have a single LinshareAPIForTesting instance ;-)